pandas/core/strings.py

```diff
@@ -1080,6 +1083,15 @@ def __iter__(self):
 
     def _wrap_result(self, result, **kwargs):
 
+        # for category, we do the stuff on the categories, so blow it up
+        # to the full series again
+        # But for some opperations, we have to do the stuff on the full values,
+        # so make it possible to skip this step as the method already did this before
+        # the transformation...
+        use_codes = kwargs.get('use_codes', True)
```

Comment:
I didn't because name wasn't a kwarg, too:

```
name = kwargs.get('name') or getattr(result, 'name', None) or self._orig.name
```

Or do I misunderstand something here?